content/PreparedStatement/PreparedStatement.facts.md

```diff
@@ -0,0 +1,62 @@
+### Facts for Prepared Statement lesson:
+
+A `prepared statement` is used to execute an SQL statement multiple times with the same or different parameter values. The execution of a prepared statement consists of using `prepare()` and `execute()` methods. 
```

Comment:
I think we need to answer `why do we need to use prepared statement` question in here also.